### PR TITLE
Many more ship name styles and words for civilians and pirates

### PR DESCRIPTION
**Pull request by Pointedstick**
Still more names! This adds a LOT of new variety to civilian and pirate ship names. With this, I feel like we're finally getting close to the point where there are enough for them, given how often you see them.

### CODE REVIEW

File: data/names.txt [hunk 1]
```diff
@@ -265,13 +308,24 @@ phrase "nouns of indeterminate quantity"
...
+		"Grass"
 
 phrase "the-nouns"
 	word
 		"the "
 	phrase
 		"singular nouns"
+		"plural nouns"
 		"space nouns"
+		"cardinal directions"
```
Comment (Pointedstick) [inline data/names.txt]: No. There is a 1 in 4 chance that "cardinal directions" will appear in "the-nouns", which themselves only have a 1 in 4 chance to appear in "things you can be of", which appear in 3 out of 12 name styles. So the actual chance of getting a cardinal direction in a ship name is 0.25^3, or 1.56%.

Comment (jafdy) [inline data/names.txt]: Point.
Over represented compared to the other "the-nouns".

Comment (Pointedstick) [inline data/names.txt]: Give it a whirl and see for yourself. I tested this extensively and didn't notice an excessive number of ships with cardinal directions in the name.

**Comment by endless-sky** [comment]
Giving this a test run:

civilian | civilian | pirate
--- | --- | ---
Ten Kings | Pious Partha | Azazel's Prince
Admiral Isiah | Kings of Heaven | Hatepunch
King's Genie | Master Darius | Khan's Poison
Sensible Sapphire | Duke of the Shockwave | Dominator of Destruction
Dukes of Heart | Prince's Ice | Dead Barrel
Champion's Cavalier | Brass Prince | Wolf of Destruction
Intrepid Alabaster | Slippery Khufu | Ghoulish Deava
Radiant Monty | Tower Sovereign | Star of Destruction
Masters and Princesses | Bruno's Spirit | Tarik's Conquest
Masters of Glass | Flood of Ouroboros | Brainhunter
Sovereign of Hands | Queen Sato | Blackbeard's King
Tower Duchess | Shimmering Journey | Balthazar's Destruction
Duchess's Masters | Adventurous Sapphire | Fog of Torment
Valiant Copper | Lilac King | Dark Baal
Sapphire Kings | Noble Idols | Corsair of Agony
Captain's Paladin | Nebula of Virtue | Antelope
Sovereign of Salvation | King of Promise | Towering Vexan
Sky of Beauty | Emperors of the Cloud | Brig of Doom
Azure Marianne | Captain's Lover | Prosperous
Lavender Tree | Sensible Quasar | Ghoulish Exile
Xiao Yi | Temple of the North | Blackbeard
Stalwart Brass | Nine Princes | Galley of Apocalypse
Saint of Golden | Waleed's Spirit | Crumbling Vexan
Saint Amwe | Twenty Men | Tankard of Domination
Henry's Sorrow | Careful Moon | Bonebat
Champion of the South | Quest of Change | Demon of Pain
Five Kings | Men and Princes | Shadow Princess
Admiral of Marigold | George Washington | Morgan le Fay
Sensible Sapphire | Riches and Dukes | Shadow Raider
Moon of Starlight | Admiral of Emperors | Foul-eater
Flood of Enlightenment | Generous Rainbow | Fallen Haekati
Dukes and Emperors | Nine Lords | Belial's Intruder
Generous Violet | Star of the Eagle | Deathrot
Unlucky Flags | Void of Lightning | God of Destruction
Lords and Hands | Darius's Cavalier | Evilsnap
Moon of Glory | One Thousand Men | Jeanne de Clisson
Means and Masters | Lion of Enlightenment | Deering
Marigold Ouroboros | Battered Emerald | Revenge
Santa Maria | Zenith Work Horse | Evening Pandora
Nine Echoes | Nymph King | Evening Cutlass
Mark Lord | Emperors of Ouroboros | Simon's Rover
Steel Admiral | Four Riches | Deadly Abaddon
Flood of Lavender | Tiamat Champion | Suffocating Ravager
Kings and Flags | Realm Lady | Grimbat
Hunk of Junk | Unlucky Brass | Wargrave
Duchess of Jade | Courageous Copper | Dreadful Naga
Tears of Fortitude | Three Kings | Baal's Hammer
Sanguine Saint | Dukes and Names | Caleuche
Indigo Lioness | Fish of Dukes | Malachai's Prince
Brazen Genevieve | Blade Mistress | Laughing Wolf
Quest of Glory | Lady Isabella | Hunter of Death
Dreams and Lords | Queen's Dancer | Edward Teach
Gray Lady | Favorite Flash | Rasputin's Tomb
Nebula of Thunder | No Gods, No Masters | Shackled Barge
Sinbad | Duchess of the Deeds | Mordred's Destruction
Champion Jaina | Emerald Pennant | Jeanne de Clisson
Unlucky Gray | Lion of Bravery | Resolute
Kings and Clouds | Empress of Emperors | Belial's Agony
Azure Names | Favorite Sapphire | La Nouvelle Trompeuse
Heart Admiral | Violet Captain | Pandora's Queen
Prince's Last Chance | Trail Empress | Wolf of Death
Santa Maria | Sovereign's Falcon | Lyubov Orlova
Shimmering Means | Shimmering Mary | Grimclaw
Twelve Whispers | Faces and Idols | Marauder of Screams
Lords and Men | Crossing Admiral | Golden Vexan
Queen of the Guardian | Durga's Dukes | Prince of Torment
Emperors and Kings | Forge Saint | Horns of Anguish
Thoughtful Beauty | Saint of Blue | T. T. Zion
Deepwater William | Black Anchor | Morningstar of Decay
Thunder Duchess | Prince's Swan | Teignmouth Electron
Gray Masters | Sun Hope | Blackbeard
Admiral of Emperors | Princes of Old | Brainhammer
Sun Jewel | Kings of the Heart | Painpunch
Nefertiti's Pride | Eight Cups | Grey Abaddon
Banner of the Sword | Unlucky Star | Antelope
Intrepid Champion | Blade of Silver | Morningstar of Fury
Lady of the Dukes | Blue Comet | Cannon of Malice
Crystal Admiral | Kings and Kings | Postillion
King of Indigo | Roses and Beasts | Jenny
White Aletheia | Generous Star | Spectre of Poison
Saint Jane | Prince's Jade | Cutlass of Blood
Noble Mistress | Beasts of Magic | Balthazar's Horns
Pious Hero | Blue Whispers | Morningstar of Madness
Masters of the West | Joyous Marigold | Reign of the Corsair
Four Kings | Western Flight | Brainrat
Prince's Surprise | Thirteen Whispers | Perdition's Princess
Sunset of Lords | Void of Heart | Bolt of Damnation
Saint of the Whispers | Endeavor | Dreadfrag
Shimmering Catalina | Saint of the Symbol | Sundered Bolt
Lion of the Silhouette | Emperors and Cups | Darkspit
Saint Sofia | Dreams and Ways | Painpit
Seven Ways | Stoic Mermaid | Bonefury
Majestic | Intrepid Admiral | Eurydice
Marigold Destiny | Union | Foulsnap
Dreams of Charity | Emperors and Kings | Sea Rasputin
Emperor Ruslan | Martin's Hauler | Lucifer's Apocalypse
Admiral Caroline | Mistresses of Kings | Samuel Axe
Zenith Explorer | Olga's Bounty | Morningstar of Domination
Ladies of Bronze | Ozeki | Lilith's Hook
Elizabeth's Falcon | Joyous Master | Charon's God

**Comment by Pointedstick** [comment]
Nice, @endless-sky. Your chart makes me realize that this change makes the male titles show up too often relative to the female ones. I've fixed this with https://github.com/endless-sky/endless-sky/pull/1787.

**Comment by Pointedstick** [comment]
This also makes me realize we need a lot more plural nouns. Probably singular nouns, too. I'll work on that as well.

**Pull request merged**